README.md

```diff
@@ -77,7 +77,7 @@ We highly appreciate you sending us a postcard from your hometown, mentioning wh
 
 ## Documentation
 
-You'll find the documentation on [https://docs.spatie.be/laravel-activitylog](https://docs.spatie.be/laravel-activitylog).
+You'll find the documentation on [https://spatie.be/docs/laravel-activitylog/v4/introduction](https://spatie.be/docs/laravel-activitylog/v4/introduction).
```

Comment:
Please keep this link `https://spatie.be/docs/laravel-activitylog` as it has the propper redirect and reduces maintenance